docs/content-services/interfaces/dialog.interface.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: Dialog Data Interface
+Added: v6.9.0
+Status: Active
+Last reviewed: 2024-05-17
+---
+
+# [Dialog Data Interface](../../../lib/content-services/src/lib/dialogs/dialog/dialog-data.interface.ts "Defined in dialog-data.interface.ts")
+
+Specifies interface for [Dialog Component](../dialogs/dialog.md).
+
+## Basic usage
+
+```ts
+interface DialogData {
+    title: string;
+    description?: string;
+    confirmButtonTitle?: string;
+    cancelButtonTitle?: string;
+    isConfirmButtonDisabled$?: Subject<boolean>;
+    isCloseButtonHidden?: boolean;
+    isCancelButtonHidden?: boolean;
+    dialogSize?: DialogSizes;
+    contentTemplate?: TemplateRef<any>;
+    actionsTemplate?: TemplateRef<any>;
+    descriptionTemplate?: TemplateRef<any>;
+    headerIcon?: string;
+}
+```
+
+## Properties
+
+| Name | Type | Default value | Description |
+| ---- | ---- | ------------- | ----------- |
+| title | `string` | `no` | It will be placed in the dialog title section. |
```

Comment:
I would suggest leaving the blank space for the default values instead of 'no', otherwise readers might thing the string `no` is the default value :)

Reply:
I did it by analogy with another component, but I absolutely agree with you :+1:
removed